### DISCUSSION TITLE
How to keep old output after reconnect the session

### DISCUSSION THREAD

**Discussion by nokiasbell** [Eugeny/tabby#8370] [Q&A] [answered]
Hi
I have upgrade tabby to version 1.0.196, and found the output will be clear after session reconnect, before the output will keep, is there a button can setup this feature?

**Comment by Eugeny** [comment, marked as answer]
It's in Settings -> SSH -> Clear terminal after connection

**Reply by nokiasbell** [reply]
Thanks for reply.
